schema.graphql

```diff
@@ -1,14 +1,14 @@
 type Application {
-  id: ID!
-  whitelisted: Boolean
-  expirationDate: String
-  owner: Bytes
-  deposit: BigInt
+    id: ID!
```

Comment:
Seems like spacing is different in this block vs. the bottom one. (we should really have auto scripts to fix things like this 😄)